Mathlib/RingTheory/Kaehler.lean

```diff
@@ -388,18 +385,42 @@ theorem KaehlerDifferential.End_equiv_aux (f : S →ₐ[R] S ⊗ S ⧸ KaehlerDi
     exact e₁.symm.trans (e.trans e₂)
 #align kaehler_differential.End_equiv_aux KaehlerDifferential.End_equiv_aux
 
-set_option maxHeartbeats 700000 in
--- Porting note: extra heartbeats are needed to infer the instance
--- Module S { x // x ∈ Ideal.cotangentIdeal (ideal R S) }
-set_option synthInstance.maxHeartbeats 200000 in
--- This has type
--- `Derivation R S (Ω[S⁄R]) ≃ₗ[R] Derivation R S (KaehlerDifferential.ideal R S).cotangentIdeal`
--- But lean times-out if this is given explicitly.
+/- After porting note: Lean is slow to synthesize theses instances.
+  Without them the endEquivDerivation' and endEquivAuxEquiv both have
+  significant timeouts. This was not the case in Mathlib 3 -/
```

Comment:
```suggestion
/- Note: Lean is slow to synthesize theses instances.
  Without them the endEquivDerivation' and endEquivAuxEquiv both have
  significant timeouts. -/
```
This basically was the case in mathlib3; at least, now that you wrote the type explicitly as `Derivation R S (Ω[S⁄R]) ≃ₗ[R] Derivation R S (KaehlerDifferential.ideal R S).cotangentIdeal`

Reply:
Perhaps it should read “slower”